examples/antarctic_slope_current_model.jl

```diff
@@ -0,0 +1,154 @@
+
+#using CairoMakie
+using Oceananigans
+using Oceananigans.Units: minute, minutes, hour
+using SeawaterPolynomials.TEOS10
+
+# This file sets up a model that resembles the Antarctic Slope Current (ASC) model in the
+# 2022 paper by Ian Eisenman
+
+arch = GPU()
+
+g_Earth = 9.80665
+
+Lx = 400000
+Ly = 450000
+Lz = 4000
+
+Nx = 400
+Ny = 450
+Nz = 70 # TODO: modify spacing if needed, 10 m at surface, 100m at seafloor
```

Comment:
Can we start with a smaller resolution? It'd be nice to test this on our laptops first